modules/upi-provisioning-new-hosts-in-a-upi-cluster.adoc

```diff
@@ -22,43 +22,145 @@ To provision bare-metal hosts to the cluster by using the BMO, you must set the
 
 . Create the `Secret` CR and the `BareMetalHost` CR.
 
-.. Save the following YAML in the `bmh.yaml` file:
+. Create a configuration file for the bare-metal node. Depending on whether you are using a static configuration or a DHCP server, use one of the following example `bmh.yaml` files, replacing values in the YAML to match your environment:
++
+[source,terminal]
+----
+$ vim bmh.yaml
+----
+* *Static configuration* `bmh.yaml`:
++
+[source,yaml]
+----
+---
+apiVersion: v1
+kind: Secret
+metadata:
+ name: openshift-worker-<num>-network-config-secret <1>
+ namespace: openshift-machine-api
+type: Opaque
+stringData:
+ nmstate: | <2>
+  interfaces: <3>
+  - name: <nic1_name> <4>
+    type: ethernet
+    state: up
+    ipv4:
+      address:
+      - ip: <ip_address> <4>
+        prefix-length: 24
+      enabled: true
+  dns-resolver:
+    config:
+      server:
+      - <dns_ip_address> <4>
+  routes:
+    config:
+    - destination: 0.0.0.0/0
+      next-hop-address: <next_hop_ip_address> <4>
+      next-hop-interface: <next_hop_nic1_name> <4>
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: openshift-worker-<num>-bmc-secret <1>
+  namespace: openshift-machine-api
+type: Opaque
+data:
+  username: <base64_of_uid> <5>
+  password: <base64_of_pwd> <5>
+---
+apiVersion: metal3.io/v1alpha1
+kind: BareMetalHost
+metadata:
+  name: openshift-worker-<num> <1>
+  namespace: openshift-machine-api
+spec:
+  online: True
+  bootMACAddress: <nic1_mac_address> <6>
+  bmc:
+    address: <protocol>://<bmc_url> <7>
+    credentialsName: openshift-worker-<num>-bmc-secret <1>
+    disableCertificateVerification: True <8>
+  rootDeviceHints:
+    deviceName: <root_device_hint> <9>
+  preprovisioningNetworkDataName: openshift-worker-<num>-network-config-secret <1>
+----
++
+--
+<1> Replace `<num>` for the machine and worker numbers of the bare-metal nodes in the `name`, `credentialsName`, and the `preprovisioningNetworkDataName` fields.
```

Comment:
> for the machine and worker numbers of the bare-metal nodes 

I don't know what this means. (Does each bare metal node have an identifying number, or does this refer to quantity??)  Will the typical user know?

Reply:
The `<num>` value is unique to the cluster and defined by the user.